app/main/views/jobs.py

```diff
@@ -256,7 +257,7 @@ def get_notifications(service_id, message_type, status_override=None):
         abort(404)
     filter_args = parse_filter_args(request.args)
     filter_args['status'] = set_status_filters(filter_args)
-    service_data_retention_days = None
+    service_data_retention_days = current_app.config.get('ACTIVITY_STATS_LIMIT_DAYS', 0)
```

Comment:
```suggestion
    service_data_retention_days = current_app.config['ACTIVITY_STATS_LIMIT_DAYS']
```
I think I would put it like this as `ACTIVITY_STATS_LIMIT_DAYS` is currently hardcoded to `7`. If it was defined by an env in the config, there would be a fallback value, probably `0` or `None`.

Reply:
Where is the `ACTIVITY_STATS_LIMIT_DAYS` config variable defined? I am wondering what the fallback would be, because if it's `None`, we could get the same problem.

Reply:
It's hardcoded to 7 at the moment
https://github.com/cds-snc/notification-admin/blob/0294af08b4c092ddd218383d816ba6186ba88197/app/config.py#L84

If it was defined with an env there would be a fallback value

Reply:
After talking to @AntoineAugusti offline, we have decided to put a programmatic default to `None` to fail fast if the configuration variable was not properly set with a default.